forc-plugins/forc-doc/src/render.rs

```diff
@@ -348,19 +350,31 @@ fn qualified_file_path(module_prefix: &Vec<String>, file_name: String) -> String
     file_path.to_str().unwrap().to_string()
 }
 
-fn doc_attributes_to_string_vec(attributes: &AttributesMap) -> String {
+fn docs_to_html(attributes: &AttributesMap) -> String {
     let attributes = attributes.get(&AttributeKind::Doc);
-    let mut attr_strings = String::new();
+    let mut docs = String::new();
+
     if let Some(vec_attrs) = attributes {
         for attribute in vec_attrs {
             for ident in &attribute.args {
-                attr_strings.push_str(ident.as_str())
+                docs.push_str(&format!("{}\n", ident.as_str()))
             }
         }
     }
```

Comment:
Maybe could be re-written like the following to omit extra allocations 

```rust
if let Some(vec_attrs) = attributes {                                                                                                                                                                                                
         for ident in vec_attrs.iter().flat_map(|attribute| &attribute.args) {
             writeln!(docs, "{}", ident.as_str());
         }
     }
```

Reply:
I'll make a follow up with this, nice!